packages/e2e/cypress/e2e/api/serviceAccounts.cy.ts

```diff
@@ -87,6 +89,75 @@ describe('Service Accounts API', () => {
         );
     });
 
+    it('Should access /groupAccesses with "org:admin" service account token', () => {
```

Comment:
new e2e test, this endpoint is only avaialble for admins "manage, org". 

We create a service account token with the org:admin scope and see if that matches.